tests/utils.py

```
@@ -0,0 +1,377 @@
+import asyncio
+import logging
+import random
+import string
+import time
+from contextlib import asynccontextmanager
+from typing import Any, Callable, Dict, List, Optional
+from unittest.mock import Mock
+
+import pytest
+from botocore.exceptions import ClientError
+from google.genai import types
```

Pylint (3.3.7)

[error] No name 'genai' in module 'google'

(E0611)